sharelatex/sharelatex-docker-image
# [misc] export all git repository revisions

### Description

There is a multi purpose shell script for the gathering of git revisions
 now.
It will extract all revisions that can be found in traversing of the
 current working directory. This effectively includes the
 overleaf/overleaf repo and any others that may be added downstream.

#### Manual Testing Performed

- I have rebuilt the image and verified that all revisions are dumped

<details>
<summary> comparison of the outputs </summary>

Notice one additional line at the top
<pre>
$ # pulled from docker hub
$ sudo docker run --rm -it --entrypoint cat sharelatex/sharelatex:2.1.1 /var/www/revisions.txt | sort
/var/www/sharelatex/chat,c521cf04c3a3317d36515178458934388cf8834c
/var/www/sharelatex/clsi,f285e08ee055e7b0fc0c81d598a61f40cfdebeb4
/var/www/sharelatex/contacts,6c142ccc7a6e75493337422fbea59b0c871a0207
/var/www/sharelatex/docstore,6c89b22d984941ca5a5c82968547ccecfb42e944
/var/www/sharelatex/document-updater,52a1f68b1b49d79493630f57d078b70ab03008da
/var/www/sharelatex/filestore,114c0dfee2f4bf1d39a5a64848e1fea7289268d9
/var/www/sharelatex/notifications,cf2499f6ea94be994da7f713f887e7f36ab0ab43
/var/www/sharelatex/real-time,87f28018ff8e743389593cc83a2cfc30154194cb
/var/www/sharelatex/spelling,fcd804349339ab07f0d3c7653203cb04fed87f72
/var/www/sharelatex/tags,eaa1886d73c67edda837096d591f6ae6f4e5fd0c
/var/www/sharelatex/track-changes,d38456380da6afdb869ecd28ff257203eeefa9c7
/var/www/sharelatex/web,828674949b6bb83b2386193db428a374d5a3e6f5
$ # locally built
$ sudo docker run --rm -it --entrypoint cat sharelatex/sharelatex:latest /var/www/revisions.txt | sort
/var/www/sharelatex,689104d93e14008e68e01b1bc9d65ba94ed669cb
/var/www/sharelatex/chat,0eec2c6d843c2133b1286ccba58afada9cbad683
/var/www/sharelatex/clsi,f285e08ee055e7b0fc0c81d598a61f40cfdebeb4
/var/www/sharelatex/contacts,6c142ccc7a6e75493337422fbea59b0c871a0207
/var/www/sharelatex/docstore,6c89b22d984941ca5a5c82968547ccecfb42e944
/var/www/sharelatex/document-updater,12df5c2e6ef27edb11bcf36e3733a4d4dab80fa6
/var/www/sharelatex/filestore,c8f869293273af3e6251f8d7bab714cea388cf96
/var/www/sharelatex/notifications,3600974e7d61a8edda9bc69af0b1dc0d9e4bae2e
/var/www/sharelatex/real-time,87f28018ff8e743389593cc83a2cfc30154194cb
/var/www/sharelatex/spelling,ca4d493343b258d303ebee351da20a643732adbe
/var/www/sharelatex/tags,bd2c8a3ad91649a2aac61d2d69e08a86de70b5ba
/var/www/sharelatex/track-changes,d38456380da6afdb869ecd28ff257203eeefa9c7
/var/www/sharelatex/web,5195c50831ad8e57504049b647ab96e4a1a6de72
</pre>
</details>
